### DISCUSSION TITLE
Feedback for course "Docker and Docker Compose"

### DISCUSSION THREAD

**Discussion by ludelafo** [heig-vd-dai-course/heig-vd-dai-course#113] [Polls]
Can you let us know what was easy and what was difficult for you during this chapter?

This will help us to improve the course and adapt the content to your needs. If we notice some difficulties, we will come back to you to help you.

You can use reactions to express your opinion on a comment! 🚀

Vous pouvez aussi nous laisser votre avis en français, il n'y a pas de souci !

**Comment by Schokiiiiiiii** [comment]
Maybe a bit dumb but in the code example below:

https://github.com/heig-vd-dai-course/heig-vd-dai-course/tree/main/04.01-docker-and-docker-compose/02-code-examples/08-docker-compose-with-volumes

It asks you to launch docker again and see a new custom html page on http://localhost:8080/. In the code example 07, you already did sth similar.

What the browser might do (and mine did) is cache the first page.

You can easily solve that by pressing ctrl + F5 or shift + ctrl + R on http://localhost:8080/.

Took me a good 5mins before understanding that there wasn't a problem with where the html file was but one with the browser \:'). This might just be me not being used to having anything on localhost ahah.

Otherwise awesome as a first time docker learner to go through those examples!

**Reply by ludelafo** [reply]
Awesome, thank you for your feedback! I'll take note of it and update the course material when I can.

Were you able to finish the practical content as well? Publishing your own application on your own GitHub account through the GitHub Container Registry?

**Reply by Schokiiiiiiii** [reply]
I actually got stuck to publishing the image on GitHub Container Repository.

I made the access token with necessary permission and tagged the image. I then tried pushing it to my repository and this is what I got:

```bash
$ docker push ghcr.io/schoki/java-ios-docker:latest
The push refers to repository [ghcr.io/schoki/java-ios-docker]
629dda862eda: Preparing 
01b8476ff375: Preparing 
6bb9f9a2dc3d: Preparing 
10dba2a44be1: Preparing 
b6b5c6883a4d: Preparing 
e5014fb88b13: Waiting 
039ba818f950: Waiting 
d67583d3af2d: Waiting 
073ec47a8c22: Waiting 
denied: permission_denied: create_package
```

I did try remaking the token, logging in again, tagging again but nothing seems to work.

**Reply by ludelafo** [reply]
Could it be related to this StackOverflow question? <https://stackoverflow.com/questions/69014742/github-denied-permission-denied-write-package>

If needed, we can organize a Teams call if you need additional help.

**Reply by Schokiiiiiiii** [reply]
Okay I finally got it.

I forgot that my name was schokiiiiiiii and not schoki (lowercase cause apparently that's required as well).

It would still put login successful so I never thought this could have been the problem.

Quite dumb while thinking back. Thanks for the help anyway!

**Reply by ludelafo** [reply]
Great, thanks for the feedback and well done for finding the solution!
